corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -106,15 +107,29 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
         var appPreview = FormplayerFrontend.currentUser.displayOptions.singleAppMode;
         var changeFormLanguage = toggles.toggleEnabled('CHANGE_FORM_LANGUAGE');
         var enablePrintOption = !menuResponse.queryKey;
+        var sidebarEnabled = toggles.toggleEnabled('SPLIT_SCREEN_CASE_SEARCH');
 
-        if (menuListView) {
+        if (sidebarEnabled && !appPreview && menuResponse.type === "query") {
+            var menuData = menusUtils.getMenuData(menuResponse);
+            menuData["triggerEmptyCaseList"] = true;
+            FormplayerFrontend.regions.getRegion('main').show(views.CaseListView(menuData));
```

Comment:
This could potentially be a subclass of `CaseListView`, right? I'm thinking of logic like [this](https://github.com/dimagi/commcare-hq/blob/16efb7fcc8f79e3aebfb7eb5117b3008026f6387/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/utils.js#L173-L180).

Reply:
ah ha yes added change [here](https://github.com/dimagi/commcare-hq/pull/33060#:~:text=%E2%80%A6-,4467c37,-Merge%20state)